src/main/java/de/samply/share/broker/rest/Monitoring.java

```diff
@@ -224,18 +226,57 @@ public Response createQuery(@HeaderParam(HttpHeaders.AUTHORIZATION) String auth,
           schema = @Schema(implementation = QueryObject.class))
           QueryObject queryObject) {
     try {
-      if (auth == null || !checkBasicAuth(auth)) {
+      if (auth == null || !checkBasicAuth(auth, "icinga.username", "icinga.password")) {
         return Response.status(Response.Status.UNAUTHORIZED).build();
       }
       return Response.ok(
           SearchController.releaseQuery(new Gson().toJson(queryObject.getCqlQueryList()),
-              queryObject.getTarget())).build();
+              queryObject.getTarget(), "Icinga Test", InquiryCriteriaType.IC_CQL)).build();
     } catch (Exception e) {
       logger.error(e);
       return Response.status(Response.Status.INTERNAL_SERVER_ERROR).build();
     }
   }
 
+  /**
+   * Create a new monitoring query.
+   * @param queryContainer the query as cql string and the target sites as list
+   * @return response if the query has been created
+   */
+  @Path("/v2/queries")
+  @POST
+  @Produces(MediaType.TEXT_PLAIN)
+  @Consumes({MediaType.APPLICATION_JSON, MediaType.TEXT_PLAIN})
+  @APIResponses({
+      @APIResponse(
+          responseCode = "200",
+          description = "ok",
+          content = @Content(
+              mediaType = MediaType.APPLICATION_JSON,
```

Comment:
Here you say, that re 200 response is in JSON and in @Produces you say, its text/plain.